src/backend/gl/src/window/egl.rs

```diff
@@ -217,16 +225,23 @@ impl hal::Instance<crate::Backend> for Instance {
         };
         let attributes = [
             egl::RENDER_BUFFER as usize,
-            egl::SINGLE_BUFFER as usize,
```

Comment:
With `SINGLE_BUFFER` only black window is displayed

Reply:
Can we make this conditional? Something like `if cfg!(target_os = "android") { ... } else { ... }`